src/beamlime/config/instruments/dream.py

```diff
@@ -167,6 +120,91 @@ def _make_dream_detectors() -> StreamLUT:
     }
 
 
+_reduction_workflow = DreamPowderWorkflow(
+    run_norm=powder.RunNormalization.monitor_integrated
+)
+
+_source_names = [
+    'mantle_detector',
+    'endcap_backward_detector',
+    'endcap_forward_detector',
+    'high_resolution_detector',
+]
+
+TotalCounts = NewType('TotalCounts', sc.DataArray)
+
+
+def _total_counts(data: DetectorData[SampleRun]) -> TotalCounts:
+    """Dummy provider for some plottable result of total counts."""
+    return TotalCounts(
+        data.hist(
+            event_time_offset=sc.linspace(
+                'event_time_offset', 0, 71_000_000, num=1000, unit='ns'
+            ),
+            dim=data.dims,
+        )
+    )
+
+
+_reduction_workflow.insert(_total_counts)
+_reduction_workflow[powder.types.DspacingBins] = sc.linspace(
+    dim='dspacing',
+    start=0.1,
+    stop=5.0,
+    num=2000,
+    unit='angstrom',
+)
+_reduction_workflow[powder.types.TwoThetaBins] = sc.linspace(
+    dim="two_theta", unit="rad", start=0.0, stop=3.1416, num=201
+)
+_reduction_workflow[powder.types.CalibrationData] = None
+_reduction_workflow = powder.with_pixel_mask_filenames(_reduction_workflow, [])
+_reduction_workflow[dream.InstrumentConfiguration] = (
+    dream.InstrumentConfiguration.high_flux
+)
+_reduction_workflow[powder.types.UncertaintyBroadcastMode] = (
+    powder.types.UncertaintyBroadcastMode.drop
+)
+_reduction_workflow[powder.types.KeepEvents[SampleRun]] = powder.types.KeepEvents[
+    SampleRun
+](False)
+
+# dream-no-shape is a much smaller file without pixel_shape, which is not needed for
+# data reduction.
+geometry_file_param = Parameter(
+    name='GeometryFile',
+    description='NeXus file containing instrument geometry and other static data.',
+    param_type=ParameterType.STRING,
+    default=str(get_nexus_geometry_filename('dream-no-shape')),
+)
+
+
+@instrument.register_workflow(
+    name='Powder reduction',
+    source_names=_source_names,
+    parameters=[geometry_file_param],
+)
+def _powder_workflow(source_name: str, GeometryFile: str) -> StreamProcessor:
+    wf = _reduction_workflow.copy()
+    wf[NeXusName[NXdetector]] = source_name
+    wf[Filename[SampleRun]] = GeometryFile
+    return StreamProcessor(
+        wf,
+        dynamic_keys=(
+            NeXusData[NXdetector, SampleRun],
+            NeXusData[powder.types.CaveMonitor, SampleRun],
+        ),
+        target_keys=(
+            powder.types.FocussedDataDspacing[SampleRun],
```

Comment:
Is it also possible to make an example where we load a Vanadium file and display a normalized result?
Maybe it's beyond the scope of this PR...

Reply:
It is possible, PRs will be accepted ;)